package.json

```diff
@@ -21,6 +21,7 @@
     "@unform/mobile": "^2.1.3",
     "axios": "^0.21.1",
     "babel-eslint": "^10.1.0",
+    "date-fns": "^2.17.0",
```

Comment:
Nesse caso eu migraria tudo para usar o `date-fns` e removeria o `moment`

Reply:
NIce. There's a issue for this... #18.. I'll take it.